src/categorical_algebra/Chase.jl

```diff
@@ -352,19 +352,20 @@ end
 A collage of a functor is a schema encoding the data of the functor
 It has the mapping data in addition to injections from the (co)domain.
 """
-function collage(F::FinFunctor)
+function collage(F::FinFunctor,obs=ob_generators(dom(F)))
```

Comment:
IMO, it's usually makes for clearer code to use keyword arguments with a clear keyword rather than optional positional arguments. So this would be

```julia
function collage(F::FinFunctor; objects=ob_generators(dom(F))
```